NCEP/Readme.md

```diff
@@ -11,15 +14,27 @@ The NCEP GDAS Data Processing Utility is a Python script designed to download Gl
 
 ## Overview
 
-The National Centers for Environmental Prediction (NCEP) provides GDAS data that can be used for weather prediction and analysis. This utility simplifies the process of downloading GDAS data, extracting relevant variables, and converting it into a format compatible with the GraphCast weather prediction system.
+The National Centers for Environmental Prediction (NCEP) provides GDAS data that can be used for weather prediction and analysis. This repo simplifies the process of downloading GDAS data, extracting relevant variables, and converting it into a format compatible with the GraphCast weather prediction system. In addition, it automates running GraphCast with GDAS inputs on the AWS machine.
 
-## Prerequisites
+## Prerequisites and Installation
+
+To install the package, run:
+
+```bash
+conda create --name mlwp python=3.10
```

Comment:
I assume the prerequisites are available under ca-ufs-cpldcld project?

Reply:
yes I did install everything on AWS cloud, just need to activate `mlwp` environment

Reply:
Perfect, thanks!